src/renderer/src/views/shared_components/UpsellComponent.vue

```diff
@@ -0,0 +1,253 @@
+                    <div class="col-md-4 flex justify-between items-center">
+                        <h1 class="text-center text-nowrap">
+                            Support <img src="/assets/wordmark.svg" class="cyd-wordmark" alt="Cyd">!
+                        </h1>
+                    </div>
+                    <div class="col-md-8 flex align-items-start">
+                        <p class="text-start text-muted small">
+                            Cyd is <a href="#" @click="openSourceClicked">open source</a> and made by a <a href="#"
+                                @click="collectiveClicked">worker-owned collective</a>. Here are ways to support us:
+                        </p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-5 upsell-col">
+                        <div class="card premium-card text-center" @click="premiumClicked">
+                            <div class="card-header">
+                                Upgrade to Premium
+                            </div>
+                            <div class="card-body small">
+                                <p>Unlock features like deleting DMs and migrating tweets to Bluesky</p>
+                                <p class="fw-bold text-center mb-0 action-text">
+                                    <template v-if="!userAuthenticated">
+                                        Sign in to start
+                                    </template>
+                                    <template v-else>
+                                        Upgrade to Premium
+                                    </template>
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-5 upsell-col">
+                        <div class="card text-center" @click="teamsClicked">
+                            <div class="card-header">
+                                Cyd for Teams
+                            </div>
+                            <div class="card-body small">
+                                <p>Get Cyd Premium for everyone at your organization or company</p>
+                                <p class="fw-bold text-center mb-0 action-text">
+                                    <template v-if="!userAuthenticated">
+                                        Sign in to start
+                                    </template>
+                                    <template v-else>
+                                        Start a Team
+                                    </template>
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-5 upsell-col">
+                        <div class="card text-center" @click="donateClicked">
```

Comment:
I feel like these should be buttons instead of div to make them keyboard focusable. Right now they are not focusable from keyboard. But I am also not a huge fan of adding structured data inside a button. So maybe the "Click to donate" should be a button, so that at least something is focusable through keyboard?

Reply:
I changed the whole `.card` divs to be buttons instead, and confirmed they can be focused with the keyboard. I agree it's not ideal to put structure data inside buttons (though I did notice that X and Facebook do it lol). But I think it's better this way, since the whole card is the clickable element, not just the text at the bottom.

Reply:
X and Facebook are very bad in their UI so I won't consider them an inspiration. I would probably use `<span>` instead of `<div>`, since that is more corect. but I can do like a HTML validation and accessibility concerns pass on the entire tool and fix or catch these then.